DOCUMENTATION.md

```diff
@@ -564,6 +565,31 @@ const client = await Client.newHazelcastClient(cfg);
+| hazelcast.client.cloud.url                             | https://coordinator.hazelcast.cloud | string          | Hazelcast Cloud URL.                                                                                                                                                                                                                                                                                                                                                                                                | <!-- markdown-link-check-disable-line -->
+| hazelcast.client.heartbeat.interval                    | 5000                                | number          | Frequency of the heartbeat messages sent by the clients to members.                                                                                                                                                                                                                                                                                                                                                 |
+| hazelcast.client.heartbeat.timeout                     | 60000                               | number          | Timeout for the heartbeat messages sent by the client to members. If no messages pass between the client and member within the given time via this property in milliseconds, the connection will be closed.                                                                                                                                                                                                         |
+| hazelcast.client.invocation.retry.pause.millis         | 1000                                | number          | Pause time between each retry cycle of an invocation in milliseconds.                                                                                                                                                                                                                                                                                                                                               |
+| hazelcast.client.invocation.timeout.millis             | 120000                              | number          | Period, in milliseconds, to give up the invocation when a member in the member list is not reachable.                                                                                                                                                                                                                                                                                                               |
```

Comment:
There are also other scenarios where we use this property. From the reference manual
```
You can set a timeout for retrying the operations sent to a member. This can be provided by using the property hazelcast.client.invocation.timeout.seconds in ClientProperties. The client retries an operation within this given period, of course, if it is a read-only operation or you enabled the redoOperation as stated in the above paragraph. This timeout value is important when there is a failure resulted by either of the following causes:

Member throws an exception.

Connection between the client and member is closed.

Client’s heartbeat requests are timed out.
```

Reply:
Why this section does not include them? https://docs.hazelcast.com/imdg/4.2/clients/java.html#client-system-properties

Reply:
opened this one https://github.com/hazelcast/hz-docs/issues/94